ceph-dev-new-build/build/build_rpm

```diff
@@ -187,7 +187,8 @@ fi
 
 # XXX perhaps use job parameters instead of literals; then
 # later stages can also use them to compare etc.
-if [[ $CI_CONTAINER == "true" && $DISTRO == "centos" && $FLAVOR == "default" ]] ; then
+# build container image that supports building crimson-osd
+if [[ $CI_CONTAINER == "true" && $DISTRO == "centos" && $FLAVOR == "crimson" ]] ; then
```

Comment:
We still want containers for `default` and `centos8`, right?  If so, you should just change this to:

```
if [[ $CI_CONTAINER == "true" && $DISTRO == "centos" && $FLAVOR != "notcmalloc" ]] ; then
```

Reply:
I was concerned do we want to have both container builds everytime, but since containers cache there penultimate builds, this won't cost much I suppose. Updated!